src/together/resources/finetune.py

```diff
@@ -617,10 +629,11 @@ def download(
             if checkpoint_type == DownloadCheckpointType.DEFAULT:
                 checkpoint_type = DownloadCheckpointType.MERGED
 
-            if checkpoint_type == DownloadCheckpointType.MERGED:
-                url += f"&checkpoint={DownloadCheckpointType.MERGED.value}"
-            elif checkpoint_type == DownloadCheckpointType.ADAPTER:
-                url += f"&checkpoint={DownloadCheckpointType.ADAPTER.value}"
+            if checkpoint_type in {
+                DownloadCheckpointType.MERGED,
+                DownloadCheckpointType.ADAPTER,
+            }:
+                url += f"&checkpoint={checkpoint_type.value}"
             else:
                 raise ValueError(
                     f"Invalid checkpoint type for LoRATrainingType: {checkpoint_type}"
```

Comment:
I think this error will never be reached because of the check in lines 614:620, so we can delete this.

Reply:
I think it's mostly for safety to ensure that no invalid values can occur past this point, given the potential inconsistency with LoRA. I'd keep this as is

Reply:
sgtm